### PR TITLE
Install Tailwind + Mix and add basic mobile view

### LINKED ISSUE #5: Mobile view
At the moment, the site isnt responsive and can't be viewed properly on mobile. It would be nice to have a mobile/tablet version.

### PR DESCRIPTION
**Pull request by sjardim**
Fixes #5  and #3

### CODE REVIEW

**Comment by sjardim** [comment]
Stack things and improves the usage of the available space on mobile views. 

CleanShot 2022-05-21 at 18 26 02@2x

**Comment by ash-jc-allen** [comment]
Hey @sjardim ! Thanks for this PR, I think it looks great!

For the time being, I'm going to close this because the mobile view was already added in #20. Someone else already has an open PR to add Vite instead of Mix (#19), but if that PR doesn't end up getting merged, I'll use this one instead.

I like how you've split out the cards into their own component file. If you were open to pulling down the latest version of `master` and doing that again, I'd be more than happy to merge it in 😄

File: resources/js/app.js [hunk 1]
```diff
@@ -1 +1 @@
-require('./bootstrap');
```
Comment (ordago) [inline resources/js/app.js:1]: @sjardim out of curiosity, why did you remove this line?

Comment (sjardim) [reply resources/js/app.js:1]: For this PR in case, I should have not removed it, my bad! 
But I usually do it when it's not needed.
